src/iso.css

```diff
@@ -12,3 +12,9 @@
 .ic-cubes .calendar-graph {
   display: none ! important;
 }
+
+@media screen and (max-width: 1130px) {
```

Comment:
This was decided on rather arbitrarily. I simply trial and error tested to make sure this wouldn't clip the graph if there was a high contribution day under it.

<img width="681" alt="Screen Shot 2020-12-11 at 11 14 23 PM" src="https://user-images.githubusercontent.com/13720087/101973497-a32c6c80-3c06-11eb-9eba-8a7d06de2798.png">